### PR TITLE
[Icon] Add PersonSegment

### PR DESCRIPTION
**Pull request by heyjoethomas**
### WHY are these changes introduced?

Adds `PersonSegment` icon for Customer Segments

### How to 🎩

🖥 [Local development instructions](https://github.com/Shopify/polaris/blob/main/README.md#install-dependencies-and-build-workspaces)
🗒 [General tophatting guidelines](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting.md)
📄 [Changelog guidelines](https://github.com/Shopify/polaris/blob/main/.github/CONTRIBUTING.md#changelog)

### 🎩 checklist

- [ ] Tested a [snapshot](https://github.com/Shopify/polaris/blob/main/documentation/Releasing.md#-snapshot-releases)
- [ ] Tested on [mobile](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting.md#cross-browser-testing)
- [ ] Tested on [multiple browsers](https://help.shopify.com/en/manual/shopify-admin/supported-browsers)
- [ ] Tested for [accessibility](https://github.com/Shopify/polaris/blob/main/documentation/Accessibility%20testing.md)
- [ ] Updated the component's `README.md` with documentation changes
- [ ] [Tophatted documentation](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting%20documentation.md) changes in the style guide

### CODE REVIEW

File: polaris-icons/icons/PersonSegmentIcon.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Person Segment
```
Comment (sam-b-rose) [inline polaris-icons/icons/PersonSegmentIcon.yml]:
```suggestion
name: Person segment
```

File: polaris-icons/icons/PersonSegmentIcon.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+description: Used for customer segments
+keywords:
+  - Person
+  - Customer
+  - Segments
+  - Customer Segment
+
```
Comment (sam-b-rose) [inline polaris-icons/icons/PersonSegmentIcon.yml]:
```suggestion
```

File: .changeset/sixty-sheep-double.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris-icons': minor
+---
+
+Add PersonSegment icon
```
Comment (sam-b-rose) [inline .changeset/sixty-sheep-double.md]:
```suggestion
Added `PersonSegment` icon
```

**Review by sam-b-rose — APPROVED** [review]
Looks great! Just had a few formatting and case convention suggestions.

**Comment by alex-page** [comment]
I will need a 16x16 variant for https://github.com/Shopify/admin-ui-foundations/tree/main/admin-ui-icons

**Pull request merged**